projects/plugins/jetpack/package.json

```diff
@@ -15,15 +15,15 @@
 	"author": "Automattic",
 	"scripts": {
 		"add-textdomain": "wpi18n addtextdomain --textdomain=jetpack --glob-pattern='!(node_modules|tests|tools|vendor){*.php,**/*.php}'",
-		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-client && yarn build-php && yarn build-extensions && yarn build-search && yarn build-asset-cdn-json",
+		"build": "yarn install-if-deps-outdated && yarn clean && node ../../../tools/cli/bin/jetpack build packages && yarn build-client && yarn build-php && yarn build-extensions && yarn build-search && yarn build-asset-cdn-json",
 		"build-asset-cdn-json": "php tools/build-asset-cdn-json.php",
 		"build-client": "gulp",
 		"build-color-schemes-wpcom": "GULP_ENV=wpcom gulp sass:color-schemes",
-		"build-concurrently": "yarn install-if-deps-outdated && yarn clean && yarn concurrently 'yarn build-client' 'yarn build-php' 'yarn build-extensions' 'yarn build-search' && yarn build-asset-cdn-json",
+		"build-concurrently": "yarn install-if-deps-outdated && yarn clean && yarn concurrently 'node ../../../tools/cli/bin/jetpack build packages' 'yarn build-client' 'yarn build-php' 'yarn build-extensions' 'yarn build-search' && yarn build-asset-cdn-json",
 		"build-extensions": "webpack --config ./tools/webpack.config.extensions.js",
 		"build-php": "composer install --ignore-platform-reqs",
-		"build-production": "yarn distclean && yarn install --production=false && yarn build-production-client && yarn build-production-php && yarn build-production-extensions && yarn build-production-search && yarn build-asset-cdn-json",
-		"build-production-concurrently": "yarn distclean && yarn install --production=false && yarn concurrently 'yarn build-production-client' 'yarn build-production-php' 'NODE_ENV=production yarn build-extensions' 'NODE_ENV=production yarn build-search' && yarn build-asset-cdn-json",
+		"build-production": "yarn distclean && yarn install --production=false && node ../../../tools/cli/bin/jetpack build packages --production && yarn build-production-client && yarn build-production-php && yarn build-production-extensions && yarn build-production-search && yarn build-asset-cdn-json",
+		"build-production-concurrently": "yarn distclean && yarn install --production=false && node ../../../tools/cli/bin/jetpack build packages --production && yarn concurrently 'yarn build-production-client' 'yarn build-production-php' 'NODE_ENV=production yarn build-extensions' 'NODE_ENV=production yarn build-search' && yarn build-asset-cdn-json",
```

Comment:
I don't really like these changes. Jetpack's build step really should not be reaching out to other parts of the monorepo to try to build things.

In particular, the changes to build-production-concurrently are going to be a useless waste of time during the CI Build job as the monorepo packages (1) were already built and (2) Jetpack will be loading them from the built copies in /tmp rather than the monorepo versions anyway.

If we want a smart build command that tries to build dependencies, perhaps instead the CLI should use the [dependency-finding script I'm adding in another PR](https://github.com/Automattic/jetpack/blob/5691e35cd07f8f7f059d32fc393d446a4d4d2e50/.github/files/find-project-deps.php) and ensure the dependencies are built before building the requested project. Bonus if it can ensure the dependencies are built _without_ blindly rebuilding them each time, and if we can have the CI Build job pass a flag to tell it not to try to build dependencies.

Reply:
That's fair. I removed that from this PR so we can discuss/land the CLI changes specifically.